Makefile

```diff
@@ -227,7 +227,9 @@ docs: build examples $(MDOX)
 check-docs: ## checks docs against discrepancy with flags, links, white noise.
 check-docs: build examples $(MDOX)
 	@echo ">> checking formatting and local/remote links"
-	PATH=${PATH}:$(GOBIN) $(MDOX) fmt --check -l --links.localize.address-regex="https://thanos.io/.*" --links.validate.config-file=$(MDOX_VALIDATE_CONFIG) $(MD_FILES_TO_FORMAT)
+	PATH=${PATH}:$(GOBIN) $(MDOX) fmt $(MD_FILES_TO_FORMAT)
```

Comment:
Why did we drop `--check -l --links.localize.address-regex="https://thanos.io/.*" --links.validate.config-file=$(MDOX_VALIDATE_CONFIG)` from here? :thinking:

Reply:
:facepalm: Thanks for catching this, I guess I removed it accidentally during some testing. Fixing it right now. It looks like we'll need to drop the `--check` flag still, since otherwise the `mdox` command fails (due to diff in white space handling). However, that should be fine, since below we are checking for clean work tree, which would catch any formatting issues.

Reply:
ping @GiedriusS - hopefully now all is good, the PR might need one more re-run since doc check failed on a link check...

Reply:
Restarting 👍

Reply:
I found the link it was moved to, made the change, hopefully now it succeeds.

Reply:
```
>> detecting white noise
cannot 'run make docs and commit changes': you have unstaged changes.
M	docs/proposals-accepted/202108-more-granular-query-performance-metrics.md
```

mhm ((:

Reply:
It never ends, does it :smirk:. Fixed now, the formatting also insisted on replacing `_` with `*` in `CHANGELOG.md` so I added that as well.